dags/gmail_label_data_import_pipeline.py

```diff
@@ -0,0 +1,155 @@
+# Note: DagBag.process_file skips files without "airflow" or "DAG" in them
+
+import os
+import logging
+from datetime import timedelta
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
+from google.cloud import bigquery
+
+from data_pipeline.utils.data_store.bq_data_service import (
+    create_table_if_not_exist,
+    load_file_into_bq,
+)
+
+from data_pipeline.utils.data_store.s3_data_service import (
+    download_s3_json_object,
+)
+
+from data_pipeline.utils.dags.data_pipeline_dag_utils import (
+    create_dag,
+    create_python_task
+)
+
+from data_pipeline.gmail_data.get_gmail_data_config import (
+    GmailGetDataConfig
+)
+
+from data_pipeline.utils.pipeline_file_io import (
+    get_yaml_file_as_dict
+)
+
+from data_pipeline.gmail_data.get_gmail_data import (
+    connect_to_email,
+    get_label_list,
+    write_dataframe_to_file,
+    # get_link_message_thread,
+)
+
+USER_ID_ENV_NAME = "GMAIL_DATA_USER_ID"
+GMAIL_DATA_CONFIG_FILE_PATH_ENV_NAME = "GMAIL_DATA_CONFIG_FILE_PATH"
+DEPLOYMENT_ENV_NAME = "DEPLOYMENT_ENV"
```

Comment:
I would perhaps keep it `DEPLOYMENT_ENV_ENV_NAME` just for consistency (not an ideal name but it was mean "env name" of the "deployment env").

Just a comment: There seem to be a bit of duplication across the DAGs, like this constant. Worth refactoring. We currently have https://github.com/elifesciences/data-hub-issues/issues/7 which may partially cover it, although in this case its duplicated within the same repo.